homeassistant/components/myuplink/icons.json

```diff
@@ -0,0 +1,47 @@
+{
+  "entity": {
+    "binary_sensor": {
+      "elect_add": {
+        "default": "mdi:electric-switch",
+        "state": {
+          "on": "mdi:electric-switch-closed",
+          "off": "mdi:electric-switch"
```

Comment:
A specific `off` case shouldn't be needed, since it's the same as the `default`.